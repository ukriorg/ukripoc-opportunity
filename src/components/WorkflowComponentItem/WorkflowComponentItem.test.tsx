import React from "react";
import renderer from "react-test-renderer";
import { WorkflowComponentItem } from "./index";

const component = {
    id: "1",
    rank: 0,
    openApplication: "",
    closeApplication: "",
    __typename: "Application"
};

const deleteListing = () => {};

describe("WorkflowComponentItem", () => {
    it("renders correctly", () => {
        const tree = renderer.create(    
            <WorkflowComponentItem
                component={component}
                deleteListing={deleteListing} />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

