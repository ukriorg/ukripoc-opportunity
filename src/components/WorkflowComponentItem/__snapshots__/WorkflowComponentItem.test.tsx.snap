// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkflowComponentItem renders correctly 1`] = `
<div
  className="sc-ifAKCX iNkSEI"
>
  <div
    className="sc-bdVaJa gkOpjT"
  >
    <div
      className="sc-bwzfXH dkijAd"
    >
      <a
        href="/component/Application/1"
        onClick={[Function]}
      >
        Application
      </a>
    </div>
    <div
      className="sc-bwzfXH iXLolC"
    >
      <button
        onClick={[Function]}
      >
        Remove
      </button>
    </div>
  </div>
</div>
`;
